### PR TITLE
[1LP][RFR] map function for entities

### PR DESCRIPTION
**Pull request by izapolsk**
There are cases when we need to do something with several entities. Usually, it is just entity.check() operation. This PR should add such function

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit dc86b6cc706b06cdf31e95eb1bf21e8b3c8dcb90:

`widgetastic_manageiq/__init__.py`:
- :bangbang: Line 3067:1: **E303** *too many blank lines (3)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit f23cda241c970647088f572a49f3910657f5aaef:

`widgetastic_manageiq/__init__.py`:
- :bangbang: Line 3068:1: **E303** *too many blank lines (4)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: widgetastic_manageiq/__init__.py [hunk 1]
```diff
@@ -2801,10 +2801,15 @@ def __getattr__(self, name):
...
     def __str__(self):
-        return str(self._get_existing_entity())
+        return str(self.__repr__())
 
     def __repr__(self):
-        return repr(self._get_existing_entity())
+        try:
+            return repr(self._get_existing_entity())
+        except NoSuchElementException:
+            return '{c} name {n}, id {id}'.format(c=self.__class__,
```
Comment (RonnyPfannschmidt) [inline widgetastic_manageiq/__init__.py]: **required** for general consistency with non-evaluatable repr functions this should use `<>` around the repr string

File: widgetastic_manageiq/__init__.py [hunk 2]
```diff
@@ -2801,10 +2801,15 @@ def __getattr__(self, name):
...
     def __str__(self):
-        return str(self._get_existing_entity())
+        return str(self.__repr__())
 
     def __repr__(self):
-        return repr(self._get_existing_entity())
+        try:
+            return repr(self._get_existing_entity())
+        except NoSuchElementException:
+            return '< {c} name {n}, id {id} >'.format(c=self.__class__,
```
Comment (mshriver) [inline widgetastic_manageiq/__init__.py]: **Optional**
Consider wrapping `n` and `id` in quotes, so when they're empty strings is obvious there's an empty value there.

**Pull request merged**